samples/zephyr-hello/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
Why different from zephyr demo above? Better be one demo.

Reply:
I was mostly trying to preserve the original effort of @msink as much as possible. I don't particularly care which demo is used if only one but I would point out that this new demo is targeted at the QEMU emulator while the other requires an actual board to run on.

Reply:
Please converge to one demo in two modes, two demos would be misleading.